docs/reference/migration/migrate_8_0/search.asciidoc

```diff
@@ -173,5 +173,15 @@ If you query date fields without a specified `format`, check if the values in yo
 actually meant to be milliseconds-since-epoch and use a numeric value in this case. If not, use
 a string value which gets parsed by either the date format set on the field in the mappings or
 by `strict_date_optional_time` by default.
-
 ====
+
+ .The `geo_bounding_box` query's `type` parameter has been removed.
+ [%collapsible]
+ ====
+ *Details* +
+ The `geo_bounding_box` query's `type` parameter was deprecated in 7.14.0 and has
+ been removed in 8.0.0. This parameter is a no-op and has no effect on the query.
+ *Impact* +
+ Discontinue use of the `type` parameter. `geo_bounding_box` queries that include
+ this parameter will return an error.
+ ====
```

Comment:
The indentation is causing this to render oddly.
<img width="769" alt="Screen Shot 2021-06-24 at 9 03 25 AM" src="https://user-images.githubusercontent.com/40268737/123267546-23dbf780-d4cb-11eb-925b-4bd1a9c558f4.PNG">

I'd remove the leading space to fix it.
```suggestion

.The `geo_bounding_box` query's `type` parameter has been removed.
[%collapsible]
====
*Details* +
The `geo_bounding_box` query's `type` parameter was deprecated in 7.14.0 and has
been removed in 8.0.0. This parameter is a no-op and has no effect on the query.

*Impact* +
Discontinue use of the `type` parameter. `geo_bounding_box` queries that include
this parameter will return an error.
====
```